chromium_src/chrome/browser/ui/views/bubble/bubble_contents_wrapper.cc

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include <utility>
+
+#include "chrome/browser/ui/views/bubble/bubble_contents_wrapper.h"
+#include "components/sessions/content/session_tab_helper.h"
+#include "content/public/browser/web_contents.h"
+#include "content/public/browser/web_contents_delegate.h"
+#include "content/public/browser/web_contents_observer.h"
+
+// Trezor library opens a popup window to connect with a device.
+// The buble delegate doesnt allow to open popups and we use
+// the Browser window delegate to redirect opening new popup content
+// to the Browser delegate instead of default one.
+// In order to close all popups we also save tab ids of each opened popup window
+// and close all with the bubble together.
+#define RenderViewHostChanged                                                \
+  SetWalletDelegate(                                                         \
```

Comment:
We shouldn't have a method called `SetWalletDelegate` on BubbleContentsWrapper. This isn't a wallet delegate anyway, it's WebContentsDelegate so if anything it should be `SetWebContentsDelegate`